gh_index_issues_to_dts_portal.py

```diff
@@ -64,6 +67,18 @@ def find_knack_record_by_issue(knack_records, issue_number):
     return None
 
 
+def are_timestamps_different(knack_timestamp, issue_timestamp):
+    """
+    Returns true if the stored comment timestamp in knack differs from the issues timestamp
+    """
+    if not knack_timestamp:
+        if issue_timestamp:
+            return True
+        else:
+            return False
+    return knack_timestamp["date"] != issue_timestamp.strftime("%m/%d/%Y")
```

Comment:
i am trying to think if there's a timezone bug here. i assume `issue_timestamp` is in UTC. and knack doesn't really care what timezone you use. `knackpy` will convert the timestamp to UTC when it loads the data into memory, but i think that only applies to the `knack_timestamp["unix_timestamp"]` property. 

so it looks to me like this will reliably compare apples-to-apples timestamps, but the only edge case concern is that folks may see in Knack that they commented one day ahead of when they actually commented, since here you are saving the calendar date of the timestamp in UTC. i think we would need to test this with a comment left after 7pm CT.  not sure it matters though 🤷

Reply:
The chances that a day-off date on a comment made after 7pm are slim. 

FWIW, we're using the date on the front end to filter down to [in-progress projects that haven’t had a status update since the previous newsletter was compiled](https://atd.knack.com/test--26-apr-2024--dts--data-and-technology-services-portal#current-projects/?view_395_filters=%7B%22match%22%3A%22and%22%2C%22rules%22%3A%5B%7B%22field%22%3A%22field_649%22%2C%22operator%22%3A%22contains%22%2C%22value%22%3A%22progress%22%2C%22field_name%22%3A%22Pipeline%22%7D%2C%7B%22match%22%3A%22and%22%2C%22field%22%3A%22field_689%22%2C%22operator%22%3A%22is%20before%22%2C%22value%22%3A%7B%22date%22%3A%2202%2F14%2F2024%22%7D%2C%22field_name%22%3A%22Comment%20date%22%7D%5D%7D&view_395_page=1).

Reply:
I am comparing on comment text now